src/models/network-v2.ts

```diff
@@ -252,32 +252,43 @@ export class Network_v2 extends Model<Network_v2Methods> implements Deployable {
    * @param percentageNeededForDispute percentage is per 10,000; 3 = 0,0003
    */
   async changePercentageNeededForDispute(percentageNeededForDispute: number) {
-    return this.sendTx(this.contract.methods.changeNetworkParameter(this.Params.percentageNeededForDispute, 
-                                                                    percentageNeededForDispute * this.divisor));
+    return this.sendTx(this.contract.methods.changeNetworkParameter(this.Params.percentageNeededForDispute,
+                                                                    this.toPercentage(percentageNeededForDispute)));
   }
 
   /**
    * @param mergeCreatorFeeShare percentage is per 10,000; 3 = 0,0003
    */
   async changeMergeCreatorFeeShare(mergeCreatorFeeShare: number) {
     return this.sendTx(this.contract.methods.changeNetworkParameter(this.Params.mergeCreatorFeeShare, 
-                                                                    mergeCreatorFeeShare * this.divisor));
+                                                                    this.toPercentage(mergeCreatorFeeShare)));
   }
 
   /**
    * @param proposerFeeShare percentage is per 10,000; 3 = 0,0003
    */
   async changeProposerFeeShare(proposerFeeShare: number) {
     return this.sendTx(this.contract.methods.changeNetworkParameter(this.Params.proposerFeeShare, 
-                                                                    proposerFeeShare * this.divisor));
+                                                                    this.toPercentage(proposerFeeShare)));
   }
 
   /**
    * @param oracleExchangeRate percentage is per 10,000; 3 = 0,0003
    */
   async changeOracleExchangeRate(oracleExchangeRate: number) {
     return this.sendTx(this.contract.methods.changeNetworkParameter(this.Params.oracleExchangeRate, 
-                                                                    oracleExchangeRate * this.divisor));
+                                                                    this.toPercentage(oracleExchangeRate)));
+  }
+
+  /**
+   * Transforms a number to a percentage 
+   * using the utils 'divisor' value.
+   * 
+   * @param percentage 
+   * @returns 
+   */
+  toPercentage(percentage: number) {
```

Comment:
Introduced the toPercentage method as it was quite unclear what happened inside the tests. Now this can be conveniently called from within tests. Makes it more coherent in error-prone imo.